lib/roken/mkdir.c

```diff
@@ -33,24 +33,21 @@
 
 #include <config.h>
 
-#ifdef HAVE_DIRECT_H
```

Comment:
Adding HAVE_DIRECT_H = 1 to windows/NTMakefile.config is how this would be done in the build system.